### PR TITLE
Clean up comments and tabs

### PR DESCRIPTION
**Pull request by aeoncleanse**
Cl

### CODE REVIEW

**Comment by Sheeo** [comment]
Cl?

**Comment by aeoncleanse** [comment]
I have no clue. And WHY is travis failing. Grrrr

File: lua/armordefinition.lua [hunk 1]
```diff
@@ -1,81 +1,67 @@
...
+-- File     :  /lua/armordefinition.lua
+-- Author(s):
+-- Summary  :
+-- Copyright © 2005 Gas Powered Games, Inc.  All rights reserved.
+-----------------------------------------------------------------
 
+-- Armor Type Definitions
+-- Laid out as follows:
+-- 'Armor Name'
+-- 'Damage Type - Multiplier'
```
Comment (Sheeo) [inline lua/armordefinition.lua]: This comment should be something like:

Armor type definitions are of the form:

    {
        {
          'Name',
          'DmgType Multiplier'+
        }*
    }

But honestly I prefer just having them inline, it's more readable.

**Comment by Sheeo** [comment]
This kind of work really shouldn't be targeted at the balance branch @aeoncleanse, it's unrelated to balance.

I know some of the files aren't in the develop branch, but could you split off a PR targeted at develop to clean up those, and then we'll clean up these files when they make it to develop?

**Comment by aeoncleanse** [comment]
I targeted at balance because this is exclusively a cleanup of files affected by the balance branch so far. All this does is clean up the files balance has changed so far, and no others.

**Comment by ChrisKitching** [comment]
These sorts of commits are not useful: they clutter `git blame` and don't add any actual value.

Do this as you go along to code you touch. You can also do this to GPG-files in the commit you add them with.

**Pull request closed without merging**